packages/core/src/sharedWishlists/redux/selectors.js

```diff
@@ -0,0 +1,151 @@
+import * as fromSharedWishlistReducer from './reducer';
+import { createSelector } from 'reselect';
+import { getEntity, getProduct } from '../../entities/redux/selectors';
+
+/**
+ * Retrieves current user's shared wishlist id.
+ *
+ * @function
+ * @memberof module:sharedWishlists/selectors
+ *
+ * @param {object} state - Application state.
+ *
+ * @returns {string} Universal idenfier of the shared wishlist.
+ *
+ * @example
+ * import { getSharedWishlistId } from '@farfetch/blackout-redux';
+ *
+ * const mapStateToProps = state => ({
+ *    sharedWishlistId: getSharedWishlist(state)
+ * });
+ */
+export const getSharedWishlistId = state =>
+  fromSharedWishlistReducer.getResult(state.sharedWishlist);
+
+/**
+ * Retrieves the error state of the current user's shared wishlist.
+ *
+ * @function
+ * @memberof module:sharedWishlists/selectors
+ *
+ * @param {object} state - Application state.
+ *
+ * @returns {object|undefined} Error information, `undefined` if there are no errors.
+ *
+ * @example
+ * import { getSharedWishlistError } from '@farfetch/blackout-redux';
+ *
+ * const mapStateToProps = state => ({
+ *    error: getSharedWishlistError(state)
+ * });
+ */
+export const getSharedWishlistError = state =>
+  fromSharedWishlistReducer.getError(state.sharedWishlist);
+
+/**
+ * Retrieves the loading status of the shared wishlist.
+ *
+ * @function
+ * @memberof module:sharedWishlists/selectors
+ *
+ * @param {object} state - Application state.
+ *
+ * @returns {boolean} Loading status of the shared wishlist.
+ *
+ * @example
+ * import { getSharedWishlistLoading } from '@farfetch/blackout-redux';
+ *
+ * const mapStateToProps = state => ({
+ *    isLoading: getSharedWishlistLoading(state)
+ * });
+ */
+export const getSharedWishlistLoading = state =>
```

Comment:
This should follow the same naming convention as v2, which is `isSharedWishlistLoading`. Check if any selector does not have the same name as in v2 and change it please.